source/lib/dynamical_systems/src/Circular.cpp

```diff
@@ -11,6 +11,7 @@ Circular::Circular(const StateRepresentation::CartesianState& center,
                                                                                                       center.get_name(),
                                                                                                       center.get_reference_frame()))),
     planar_gain_(std::make_shared<StateRepresentation::Parameter<double>>("gain", gain)),
```

Comment:
gain parameter should be renamed to planar_gain parameter then. Parameters are mostly called from command line so I am ok with changing them. It should not break any behaviors

Reply:
ok, this is what i was unsure about so I didn't want to break existing behaviours. good to know, will fix.